docs/src/components/motion/speeds.module.css

```diff
@@ -1,14 +1,14 @@
 .speeds {
   border-radius: 12px;
-  background: var(--psColorsBackgroundLight2);
+  background: var(--appColorsBackgroundMed);
   padding: var(--psLayoutSpacingMedium);
 }
 .speed {
   display: grid;
   grid-template-columns: 64px 1fr;
   grid-gap: var(--psLayoutSpacingMedium);
   grid-template-areas: 'icon vars' 'labels labels';
-  color: var(--psColorsBackgroundDark2);
```

Comment:
Was this color wrong to begin with?

Reply:
It's a background color. Semantically, it's just not what we use bg colors for.  It could have been that way in the mockup, probably was. Changed it so that we use textIcon colors for text instead.